Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,611 @@
+in case of desperation.
+
+1. Use the [`itk::ImageSeriesReader`](https://itk.org/Doxygen/html/classitk_1_1ImageSeriesReader.html) in combination with the
+   `DicomSeriesFileNames`. For a full example on how to do this,
+   please look at the [`DicomSeriesReadImageWrite2.cxx`](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Examples/IO/DicomSeriesReadImageWrite2.cxx)
+   code.
+1. Write a MetaImage header. This is a small text file holding
+   information about the image: spacing, dimensions, pixelt type, etc.
+   This header can hold the list of DICOM files you want to read. The
+   only restriction is that the files must be uncompressed.
+   The easy way to create a MetaImage header is to use the application:
+   `MetaImageImporter` that is available in
+   [ITKApps/MetaImageImporter](https://github.com/InsightSoftwareConsortium/ITKApps/tree/master/MetaImageImporter).
+   You can also take an existing MetaImage reader and modify it in
+   order to fit your needs. Please find attached one file that you
+   can edit it with your favorite text editor.
+1. Use `MRIConvert`: Jolinda Smith from the Lewis Center for
+   Neuroimaging at the University of Oregon developed a nice
+   application that allows you to load DICOM series and export them in
+   MetaImage and Analyze format (among others). She graciously has made
+   this application publicly available at:
+   https://darkwing.uoregon.edu/~jolinda/MRIConvert/.
+1. Jeff Brubaker and Stephen R. Aylward, at the UNC  CADDLab
+   developed a DICOM query/move application called "MIND"
+   (with the motto: "MIND is not DICOM"). This application loads
+   DICOM files over the network and export them in MetaImage format.
+   This application is open source and it is available at:
+   http://www.jeffro.net/mind/ and http://caddlab.rad.unc.edu/software/MIND/.
+
+### How do I write a volume as DICOM series?
+
+Use ITK in combination with GDCM: http://gdcm.sourceforge.net/.
+
+GDCM is an open source package developed by the Creatis team at
+INSA-Lyon. It is distributed under a license similar to ITK: http://gdcm.sourceforge.net/Copyright.html.
+
+GCDM uses CMake in order to configure its build process, so you will
+find a familiar setup. Once you download, configure and buid GDCM, you
+can reconfigure ITK by running CMake in your ITK build, going into the
+advanced options and enabling `USE_GDCM`.
+
+For a code example on how to use GDCM for reading and writing DICOM
+series, please refer to the source code in [`DicomSeriesReadImageWrite2.cxx`](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Examples/IO/DicomSeriesReadImageWrite2.cxx).
+
+You can always use the latest and greatest of GDCM, simply use an
+installed version of GDCM and link ITK to it using `ITK_USE_SYSTEM_GDCM`.
+
+### Which interpreted languages are supported by ITK?
+
+Currently ITK supports the following language bindings:
+
+- Python
+- Java
```

Comment:
Should be revisited: not sure if we still support Java or whether this goes through SimpleITK.

Reply:
We have Python language bindings, but Java-SWIG is not maintained.

We can point people to itk-wasm and SimpleITK.